dlp/api/DlpTest/DlpTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class DlpTest
+    {
+        private readonly string _projectId;
+        const string phone = "(223) 456-7890";
+        const string email = "gary@somedomain.org";
+        const string cc = "4000-3000-2000-1000";
+        const string inspectFilePath = "test.txt";
+        const string inspectStringValue = "hi my phone number is (223) 456-7890 and my email address is gary@somedomain.org. " +
+            "You'll find my credit card number is 4000-3000-2000-1000!";
+
+        readonly CommandLineRunner _dlp = new CommandLineRunner()
+        {
+            VoidMain = Dlp.Main,
+            Command = "Dlp"
+        };
+
+        protected ConsoleOutput Run(params string[] args)
+        {
+            string cmd = args[0].ToLower();
+            return _dlp.Run(args);
+        }
+
+        public DlpTest()
+        {
+            _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        }
+
+        private void AssertPhoneEmailCC(ConsoleOutput output, bool checkPhone, bool checkEmail, bool checkCc)
+        {
+            var results = 0;
+            if (checkPhone) results++;
+            if (checkEmail) results++;
+            if (checkCc) results++;
+            Assert.Contains($"Found {results} results", output.Stdout);
+            if (checkPhone) Assert.Contains(phone, output.Stdout);
```

Comment:
nit: having "if" as one liners is error prone, so I feel that we shouldn't have it in the sample code.
Better to write full:
if (xxx)
{
  ...
}

Reply:
Typically I have always followed the same pattern. I made an exception in this case since they are short enough to all go on a single line, whereas I think the really error prone ones are these:

```
if (condition)
    doSomething();
```

I can expand this out though for consistency's sake.